packages/gestalt/src/utils/KeyCodes.js

```diff
@@ -0,0 +1,4 @@
+// @flow strict
+export const ESCAPE_KEY_CODE = 27;
+
+export default { ESCAPE_KEY_CODE };
```

Comment:
@themindfuldev why do we need both `export const` + `export default {`?

Could we do one or the other?

Reply:
Definitely! I tried just `export const ESCAPE_KEY_CODE = 27;` but Flow insisted on `default` and I forgot to remove the first :P

Reply:
Since this will likely grow, I was thinking of something like
```
const KEY_CODES = {
  ESC: 27,
};

export default KEY_CODES;
```
What do y'all think?